google-basemap/index.ts

```diff
@@ -0,0 +1,138 @@
+import './style.css';
+import {GoogleMapsOverlay as DeckOverlay} from '@deck.gl/google-maps';
+import {colorCategories} from '@deck.gl/carto';
+import {VectorTileLayer, vectorQuerySource} from '@deck.gl/carto';
+import {Loader} from '@googlemaps/js-api-loader';
+
+const GOOGLE_MAPS_API_KEY = import.meta.env.VITE_GOOGLE_API_KEY;
+const GOOGLE_MAP_ID = import.meta.env.VITE_GOOGLE_MAP_IDS;
+const apiBaseUrl = import.meta.env.VITE_API_BASE_URL;
+const accessToken = import.meta.env.VITE_API_ACCESS_TOKEN;
+const connectionName = import.meta.env.VITE_API_CONNECTION_NAME;
```

Comment:
Remove this, we will add another PR to include connectionName in all the examples

Reply:
Do I change it in the .env like this?
```js
VITE_API_CONNECTION_NAME= # TODO, will be add another PR to include connectionName in all the examples
```

Reply:
Just remove the connection name in this example and open another PR to include the connection name in all the examples